backend/app/models/listeners.py

```diff
@@ -69,6 +69,7 @@ class EventListenerDB(Document, EventListenerBase):
     modified: datetime = Field(default_factory=datetime.now)
     lastAlive: datetime = None
     alive: Optional[bool] = None  # made up field to indicate if extractor is alive
+    active: Optional[bool] = None
```

Comment:
This should probably be just `bool` and not optional. It's either enabled or not. Default should be False.

(Same could be said about `alive` above)